docs/tutorial/dbinit.rst

```diff
@@ -35,7 +35,7 @@ just below the `connect_db` function in :file:`flaskr.py`::
     def initdb_command():
         """Initializes the database."""
         init_db()
-        print 'Initialized the database.'
+        print('Initialized the database.')
```

Comment:
Yes please.

On 2 August 2016 16:54:00 CEST, Jiri Kuncar <notifications@github.com> wrote:
>> @@ -35,7 +35,7 @@ just below the `connect_db` function in
>:file:`flaskr.py`::
>>      def initdb_command():
>>          """Initializes the database."""
>>          init_db()
>> -        print 'Initialized the database.'
>> +        print('Initialized the database.')
>
>What about using `click.echo('Initialized the database.')`?
>
>---
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/pallets/flask/pull/1973/files/86ffc0a1010f2e9f8d76593471628bbc973e9453#r73170489

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.